web/src/WifiHiddenNetworkForm.test.jsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+import { screen } from "@testing-library/react";
+import { installerRender } from "./test-utils";
+
+import WifiHiddenNetworkForm from "./WifiHiddenNetworkForm";
+
+const beforeDisplayingFn = jest.fn();
+
+jest.mock("./WifiConnectionForm", () => () => "WifiConnectionForm mock");
+
+describe("WifiHiddenNetworkForm", () => {
+  describe("when it is visible", () => {
+    it("renders the WifiConnectionForm", async () => {
```

Comment:
No need for async in `WifiHiddenNetworkForm` tests (except for the last one which tests user interaction). Please, check #313 for an example.